.travis.yml

```diff
@@ -3,6 +3,9 @@ node_js:
   - '5'
   - '4'
 
+before_install:
+  - npm install -g npm@latest
+ 
```

Comment:
Not required anymore.  At least for node 5.  By the way I think you should disable  node 4 on Travis.